src/react-apps/applications/ux-editor/src/containers/FormDesigner.tsx

```diff
@@ -222,7 +222,16 @@ class FormDesigner extends React.Component<
                 </CollapsableMenuComponent>
                 <CollapsableMenuComponent
                   header={this.props.language.ux_editor.service_logic_calculations}
+<<<<<<< HEAD
```

Comment:
Merge conflict i innsjekk?

Reply:
This branch is based on another branch that is just merged into to master. It didn't handle it very well. Working on finding all merge conflict comments.